earth_enterprise/legacy_installer/README.txt

```diff
@@ -0,0 +1 @@
+This is the obsolete installer. See ../src/installer for the scripts you want.
```

Comment:
Nit: might want to reword this a bit - perhaps explain what the legacy installer is and why it's still around in addition to pointing to the new installer.  @tst-ccamp you might have a better idea of what to say here.